common/src/main/scala/explore/components/TimeSpanView.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import explore.model.syntax.all.toHoursMinutes
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.util.TimeSpan
+import lucuma.react.common.ReactFnProps
+
+case class TimeSpanView(timespan: TimeSpan, modifiers: Seq[TagMod] = Seq.empty)
```

Comment:
I guess this will eventually go to `lucuma-ui`

Reply:
Yeah good idea I should probably put this there